rtv/templates/rtv.cfg

```diff
@@ -57,6 +57,9 @@ oauth_redirect_port = 65000
 ; Access permissions that will be requested.
 oauth_scope = edit,history,identity,mysubreddits,privatemessages,read,report,save,submit,subscribe,vote
 
+; Maximum number of columns for a comment
+max_comment_cols = 80
```

Comment:
Would you be opposed to making the default 120? I think that it maintains the readability while allowing more text to fit on the screen at once.